api/HTMLDirectoryElement.json

```diff
@@ -0,0 +1,98 @@
+{
+  "api": {
+    "HTMLDirectoryElement": {
```

Comment:
This data seems to contradict what we have in dir.json, and I suspect that data must be wrong. Because http://mdn-bcd-collector.appspot.com/tests/api/HTMLDirectoryElement involves `document.createElement('dir')`, if those tests return true then it also implies support for the element, not just the interface.

Simply pasting the same data would be fine I think, given how old all of this is and the ranges there's very little room for it to be wrong, so no need to dig deep here.